generators/app/templates/src/app/screens/Registration/index.tsx

```diff
@@ -1,89 +1,86 @@
-import React, { useCallback } from 'react';
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { useFormik } from 'formik';
 import i18next from 'i18next';
 
-import FormInput from '~components/FormInput';
-import PATHS from '~components/Routes/paths';
-import { stringArrayToObject } from '~utils/array';
+import { useLazyRequest } from '~app/hooks/useRequest';
+import { signUp } from '~services/AuthServices';
+import { UserData } from '~contexts/UserContext/reducer';
 
+import FormStep from './components/FormStep';
+import { steps, initialValues } from './constants';
+import { validate } from './validations';
 import styles from './styles.module.scss';
 
-const FIELDS = stringArrayToObject(['name', 'surname', 'email', 'password', 'confirmPassword']);
+function Registration() {
+  const history = useHistory();
+  const [currentStep, setStep] = useState(0);
 
-function RegistrationContainer() {
-  /* eslint-disable @typescript-eslint/no-unused-vars */
-  const handleSubmit = useCallback((event: React.FormEvent<Element>) => {
-    // TODO: Implement
-  }, []);
+  const handleNextStep = () => {
+    if (currentStep < steps.length - 1) {
+      setStep(currentStep + 1);
+    }
+  };
 
-  const handleInputChange = useCallback((event: React.FormEvent<Element>) => {
-    // TODO: Implement method
-  }, []);
+  const handlePrevStep = () => {
+    if (currentStep > 0) {
+      setStep(currentStep - 1);
+    }
+  };
+
+  const [, , signUpError, signUpRequest] = useLazyRequest({
+    request: signUp,
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    withPostSuccess: response => {
+      history.push('/');
+    }
+  });
+
+  const handleFormSubmit = (values: UserData) => {
+    signUpRequest(values);
+  };
+
+  const formik = useFormik({ initialValues, onSubmit: handleFormSubmit, validate });
+
+  const { handleChange, handleSubmit, values, errors, isValid } = formik;
```

Comment:
Can't you destructure this variables in the first line? Like:

```suggestion
  const { handleChange, handleSubmit, values, errors, isValid } = useFormik({ initialValues, onSubmit: handleFormSubmit, validate });
```